src/main/java/uk/gov/ch/repository/update/OverseasEntityDataRepository.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.ch.repository.update;
+
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.PagingAndSortingRepository;
+import uk.gov.ch.model.update.OverseasEntityData;
+
+public interface OverseasEntityDataRepository extends PagingAndSortingRepository<OverseasEntityData, String> {
+
+    @Query(value = "select "
+            + "CBD.email_address "
+            + "from CORPORATE_BODY CB "
+            + "inner join corporate_body_detail CBD on CB.corporate_body_id=CBD.corporate_body_id "
+            + "where "
+            + "CB.incorporation_number = ?" ,
+            nativeQuery = true)
+    OverseasEntityData getOverseasEntityData(String companyNumber);
```

Comment:
is companyNumber the same as entityNumber?

Reply:
incorporation_number is the OE company number / entity number.

Reply:
Yes, and the same as overseas entity id. There's some inconsistency going on here. My suggestion is to use 'incorporationNumber' all the way through the implementation.

Reply:
> is companyNumber the same as entityNumber?

Yes, and the same as overseas entity id. There's some inconsistency going on here. My suggestion is to use 'incorporationNumber' all the way through the implementation.